### PR TITLE
fixed bug with rendering current day prompt

### PR DESCRIPTION
**Pull request by sjlarson92**

### CODE REVIEW

File: src/components/PromptLayout.js [hunk 1]
```diff
@@ -11,13 +11,15 @@ export const PromptLayout = ({ prompts, date, updateNextDate, updatePreviousDate
...
-        data-testid="prompt"
-        prompt={prompts[date]}
-      />)
-    }
+
+      {
+          (Object.keys(prompts).length > 0 && prompts[date]) &&
+              (<Prompt
+              data-testid="prompt"
+              prompt={prompts[date]}
```
Comment (lgc13) [inline src/components/PromptLayout.js]: SPAAACES

Comment (lgc13) [reply src/components/PromptLayout.js]: tabbbb*

File: src/components/PromptLayout.js [hunk 2]
```diff
@@ -11,13 +11,15 @@ export const PromptLayout = ({ prompts, date, updateNextDate, updatePreviousDate
...
-    
-    {Object.keys(prompts).length > 0 &&
-      (<Prompt
-        data-testid="prompt"
-        prompt={prompts[date]}
-      />)
-    }
+
+      {
+          (Object.keys(prompts).length > 0 && prompts[date]) &&
```
Comment (lgc13) [inline src/components/PromptLayout.js]: is there a test for this?

Comment (sjlarson92) [reply src/components/PromptLayout.js]: there is now :)

**Pull request merged**